### ISSUE TITLE
MockResponse wrong content-length

### ISSUE THREAD

**Issue by lazar-otasevic-cif** [apollographql/apollo-kotlin#4682]
### Version

3.4.0

### Summary

this function takes wrong body length - it takes string character count `body.length` but that does not match utf8 byte-count that is `body.encodeUtf8().length` in general case, so it causes bugs with some characters inside the body:

```
  constructor(
      body: String,
      statusCode: Int = 200,
      headers: Map<String, String> = emptyMap(),
      delayMillis: Long = 0,
  ) : this(
      statusCode = statusCode,
      body = flowOf(body.encodeUtf8()),
      headers = headers + mapOf("Content-Length" to body.length.toString()),
      delayMillis = delayMillis,
  )
```

### Logs

```
(Your logs here)
```

**Comment by martinbonnin** [comment]
Hi 👋 
Can you check with the latest version (`3.7.4`) ? I'm almost sure this has been fixed there.

**Comment by martinbonnin** [comment]
My bad, it's still there, I'll fix it. In the meantime, you can use with the Builder version:

```kotlin
val response = MockResponse.Builder().body("hello world").build()
```

Is is certainly a safer bet anyways as the constructor is deprecated and will be removed at some point.

**Comment by BoD** [comment]
This has been released as part of [v3.7.5](https://github.com/apollographql/apollo-kotlin/releases/tag/v3.7.5)
